charts/timed/templates/deployment-backend.yaml

```
@@ -33,6 +33,8 @@ spec:
                   fieldPath: status.podIP
             - name: DJANGO_ALLOWED_HOSTS
               value: '{{ join "," .Values.ingress.hosts | default "localhost" }},$(THIS_POD_IP)'
+            - name: HURRICANE_REQ_QUEUE_LEN
+              value: {{ .Values.backend.hurricane.reqQueueLen }}
```

it looks like hurricane values are in settings and not directly in backend leading to the ci fail we're seeing
```suggestion
              value: {{ .Values.backend.settings.hurricane.reqQueueLen }}
```

---

Thanks! should be okay now

---

Ok thanks, that's fixed.